projects/storefrontstyles/scss/components/content/navigation/_navigation-ui.scss

```diff
@@ -360,7 +333,7 @@
         }
       }
       > nav:last-child {
-        overflow: hidden;
+        overflow: visible;
```

Comment:
it seems that we can drop this rule altogether as the calydon one was replacing the former hidden value. Visible is default however.

Reply:
This isn't changed afaik. If visible is the current/inherited state, then we can drop the rule altogether.

Reply:
mmm, sorry I might be looking at old code